React Native/Navigation.md

```diff
@@ -0,0 +1,203 @@
+Routing in mobile applications presents more then mere navigation between routes (screens). It consists of functionalities and styles which are divided across platforms. Generally, routing systems inside native applications has its own look and feel aswell as well defined UX which differs from android and iOS platforms.
```

Comment:
`has its own look and feel aswell as well defined UX which differs` should probably be
`has its own look and feel as well defined UX which differs`